dmutils/forms/dm_fields.py

```diff
@@ -0,0 +1,65 @@
+import wtforms.fields
+
+import dmutils.forms.fields
+
+from .mixins import DMFieldMixin
+
+
+__all__ = ['DMBooleanField', 'DMDecimalField', 'DMEmailField', 'DMHiddenField',
+           'DMIntegerField', 'DMRadioField', 'DMStringField', 'DMStripWhitespaceStringField']
+
+
+class DMBooleanField(DMFieldMixin, wtforms.fields.BooleanField):
+    pass
+
+
+class DMDecimalField(DMFieldMixin, wtforms.fields.DecimalField):
+    pass
+
+
+class DMEmailField(DMFieldMixin, dmutils.forms.fields.EmailField):
+    pass
+
+
+class DMHiddenField(DMFieldMixin, wtforms.fields.HiddenField):
+    pass
+
+
+class DMIntegerField(DMFieldMixin, wtforms.fields.IntegerField):
+    pass
+
+
+class DMRadioField(DMFieldMixin, wtforms.fields.RadioField):
+    '''
+    A Digital Marketplace wrapper for `wtforms.RadioField`.
+
+    The `choices` argument for the constructor should be a sequence of
+    `(value, label, description)` tuples.
+
+    The `options` property is the choices in a format suitable for the
+    frontend toolkit.
+    '''
+    def __init__(self, label=None, validators=None, choices=None, **kwargs):
+        if choices:
+            # We do this the long way rather than using a comprehension because
+            # we want to be able to accept (value, label) tuples as well.
+            self.options = []
+            for choice in choices:
+                option = {}
+                option['value'] = choice[0]
+                option['label'] = choice[1]
+                if len(choice) > 2 and choice[2]:
+                    option['description'] = choice[2]
+
+            # construct a choices argument suitable for wtforms.fields.RadioField
+            choices = [(option['value'], option['label']) for option in self.options]
```

Comment:
might be more efficient to do the following:

```
self.options = []
wtforms_choices = []
for choice in choices:
    option = {}
    option['value'] = choice[0]
    option['label'] = choice[1]
    if len(choice) > 2 and choice[2]:
        option['description'] = choice[2]
    
    wtforms_choices.append( (option['value'], option['label']) )

super().__init__(label, validators, choices=wtforms_choices, **kwargs)
```

Reply:
I'd prefer to keep the two separate as I think it's clearer to read. Is that okay with you?